index.html

```diff
@@ -304,13 +303,43 @@ <h2>A Special Time</h2><br />
       <p>“If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the
         team, “you wind up at inflation.”</p><br>
     </section>
+    <form class="form-container" action="#" method="post">
+      <fieldset>
+        <legend>comment</legend><br />
+        <label class="margin" for="User_name">Name: </label>  
+        <input class="input-field" type="text" id="User_name" name="User_name" maxlength="20" required><br><br>
+        
+        <label for="email">Email: </label>
+        <input class="input-field" type="email" id="email" name="email" maxlength="20" required><br><br>
+        
+        <label for="comment">Comment</label><br />
+        <textarea class="textfield" id="comment"  name="comment" placeholder="Type your comment here…" rows=5 maxlength="250" required></textarea>
+        <br />
+          <fieldset >
+            <legend style="margin: 0 auto;">What is your overall opinion about this article?</legend>
+              <input type="radio" id="position" name="opinion" value="position" checked>
+              <label for="position">Position</label><br>
+              <input type="radio" id="negative" name="opinion" value="negative">
+              <label for="negative">Negative</label><br>
+              <input type="radio" id="neutral" name="opinion" value="neutral">
+              <label for="neutral">Neutral</label>
```

Comment:
You have the radio button fieldset, but your "positive" option is misspelled as "position".